pymc_experimental/model_builder.py

```diff
@@ -275,10 +275,10 @@ def set_idata_attrs(self, idata=None):
 
         Examples
         --------
-        >>> import arviz as az
-        >>> idata = az.InferenceData()
-        >>> self.set_idata_attrs(idata=idata)
-        >>> assert "id" in idata.attrs
+        >>> model = YourModel(ModelBuilder)
```

Comment:
Maybe keep it consistent between `MyModel`, `YourModel`.

Reply:
My mistake, didn't notice that one. It's fixed now.